### PR TITLE
Modify rule S2711: LaYC format

### PR DESCRIPTION
**Pull request by joke1196**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S2711/python/rule.adoc [hunk 1]
```diff
@@ -1,18 +1,50 @@
...
+
+If the goal is to break out of a loop, use `break` instead of `return`.
+
+=== Code examples
+
+==== Noncompliant code example
 
+[source,python,diff-id=1,diff-type=noncompliant]
+----
+class MyClass:
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S2711/python/rule.adoc:15]: I understand we have a class here to highlight the fact that we're not in a function, but having a loop here feels pretty weird as well. Wdyt?
Would it make sense to do away with the class and pretend this is module-level code?

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
I have the feeling the code example is weird, with a function inside a class.
Not requesting changes, but if you think this can be made a bit better, please don't hesitate.
Looks good otherwise.

File: rules/S2711/python/rule.adoc (hunk 1 above)
Comment (joke1196) [reply rules/S2711/python/rule.adoc:15]: Yes, I think you are right it makes enough sense without the Class.

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**